adapters/faraday/lib/opentelemetry/adapters/faraday/adapter.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'faraday'
+require 'opentelemetry'
+
+require_relative 'middlewares/tracer_middleware'
+require_relative 'patches/rack_builder'
+
+module OpenTelemetry
+  module Adapters
+    module Faraday
+      class Adapter
+        class << self
+          attr_reader :config,
+                      :propagator,
+                      :tracer
+
+          def install(config = {})
+            @config = config
+            @propagator = OpenTelemetry.tracer_factory.rack_http_text_format
+            @tracer = config[:tracer] || default_tracer
```

Comment:
I don't think the tracer should be overridable. We should just use the one we lookup from the tracer factory.

Reply:
This was moved to a helper method and removed from config after our SIG work session.